src/pages/QuestionListsPage/QuestionLists.tsx

```diff
@@ -67,8 +66,35 @@ export default function QuestionLists(): JSX.Element {
     setModalVisibility(false);
   };
 
-  const tags = (interviewQuestions: QuestionModel[]): JSX.Element[] => {
-    return getDistinctCategories(interviewQuestions).map((category) => (
+  const showSuccessModal = (): void => {
+    const successModal = Modal.success({
+      title: `Question Set was deleted.`,
+    });
+    setTimeout(() => successModal.destroy(), 3000);
+  };
+
+  const deleteList = (id: number): void => {
+    client.deleteQuestionSet(id).then(() => {
+      setLists(lists?.filter((item) => item.questionSet?.id !== id));
+      setTimeout(() => showSuccessModal(), 500);
+    });
+  };
+
+  const showConfirmationModal = (title: string, onOk: () => any): void => {
+    Modal.warning({
+      title: `Delete ‘${title}’ Question Set?`,
+      content: 'This change cannot be undone.',
+      okText: 'Yes',
+      okButtonProps: { danger: true },
+      onOk: () => onOk(),
```

Comment:
When assigning a function to a function property that has the same parameters and return type, there is no need for an additional arrow function

```suggestion
      onOk: onOk,
```

Reply:
I supposed you can write only `onOk,` , but I had detect some problem with running delete despite of cancelation in modal window. I thought it is necessary to map it in this way. Probably it was only my local bug. ☺